apache/trafficcontrol
# Add revert certificate functionality to TPv1

Add revert certificate to ssl certificate page dropdown, allowing a user to go back to a previous certificate in the case they uploaded a failing certificate or some other mistake.


<hr/>

## Which Traffic Control components are affected by this PR?


- Traffic Portal


## What is the best way to verify this PR?


- Build and run TO and TP 
- Go to a Delivery Service's SSL page and add a certificate, you will notice there is no dropdown when the certificate is version 1 as that would simply be deleting the certificate
- Update the CSR of the cert and now it should be incremented to another version
- You can then use the dropdown to revert back to 1


## PR submission checklist
